R/viz.R

```diff
@@ -3850,14 +3910,17 @@ fancy_ideogram = function(this_sample,
 #'                                        chr_anno_dist = 4,
 #'                                        chr_select = paste0("chr", c(1:22)),
 #'                                        coding_only = FALSE,
-#'                                        from_flatfile = FALSE,
-#'                                        use_augmented_maf = FALSE)
+#'                                        from_flatfile = TRUE,
+#'                                        use_augmented_maf = TRUE)
 #'
 fancy_multisamp_ideogram = function(these_sample_ids,
                                     plot_title = "CN Segments Ideogram",
                                     plot_sub = "grch37",
                                     chr_anno_dist = 3,
                                     chr_select = paste0("chr", c(1:22)),
+                                    kompare = FALSE,
+                                    concordance = TRUE,
+                                    cn_thresholds = data.frame(cn_state = c(0:14), low_thresh = c (0:14), high_thresh = c(1:15)),
```

Comment:
I think this will be simplified once the below part converted to use pipe?

Reply:
Yes, it has been simplified, indeed :)